Teste full stack II parte Back

welitoonl opened the pull request:
# API para o teste full stack II

Não tive tempo hábil devido a fatores externos, para criar o front, então decidi enviar apenas o backend
No readme tem as instruções para rodar o código 

Backend feito utilizando:

Node com:
- typescript
- express - Para o server
- typeOrm - Para o banco de dados
- tsrynger - Para injeções de dependência
- PostgreSQL
- Docker

 Caso tivesse tempo hábil
- Documentação com Swagger
- Front: Vuejs e Nuxtjs (igual ao primeiro teste)

welitoonl committed:
Teste full stack II parte Back

Realizado o teste full stack II com o backend completo, criada a API utilizando:
- Nodejs
- express
- typeORM
- tsrynger
- postgres
- docker

guilopesbearari commented:
Olá Weliton, nesse segundo teste precisaríamos do front pois é o que completa toda a usabilidade da proposta. Até dia 12/05 é viável enviar?

Obrigado.

welitoonl commented:
Boa tarde Guilherme, tudo certo? é sim, darei continuidade ao projeto então!

Obrigado.

welitoonl committed:
Inicialização do App com Nuxt

welitoonl committed:
Desenvolvimento app front

Criado index com listagem e botões de ação, Criado pagina de criação de campo e de preenchimentos de campos

welitoonl committed:
Subindo front

guilopesbearari commented:
Olá Welliton, desculpa a demora do retorno.

Fizemos os testes e encontramos alguns problemas:
- Quando acontece qualquer erro no momento de salvar um campo, não tem nenhuma interação com o usuário;
- Não conseguimos criar as 3 opções do combo

Mas tudo bem, vamos seguir contigo para um bate papo :)

Obrigado pela participação e em breve entrarão em contato contigo.
